eventmesh-common/src/main/java/org/apache/eventmesh/common/protocol/grpc/common/EventMeshCloudEventUtils.java

```diff
@@ -47,8 +47,12 @@ public static String getEnv(CloudEvent cloudEvent) {
     }
 
     public static String getEnv(CloudEvent cloudEvent, String defaultValue) {
+        return getValue(cloudEvent, defaultValue, ProtocolKey.ENV);
+    }
+
+    private static String getValue(CloudEvent cloudEvent, String defaultValue, String protocolKey) {
         try {
-            return cloudEvent.getAttributesOrThrow(ProtocolKey.ENV).getCeString();
+            return cloudEvent.getAttributesOrThrow(protocolKey).getCeString();
         } catch (Exception e) {
             return defaultValue;
         }
```

Comment:
Maybe it would be better to put the utility method at the top or bottom of the method calls.

Reply:
Moved on top.